index.bs

```diff
@@ -437,6 +446,9 @@ do support suborigins. The following discussions discuss the three major
 cross-origin mechanisms that are relevant: <a>CORS</a>, <a>`postMessage`</a>,
 and Workers [[!WORKERS]].
 
+Issue: TODO(devd): Making things specific to XHR or CORS is weird. We should
```

Comment:
no, because once you make all fetches crossorigins, then even SRI script tag fetches are cross origin?